src/README.md

```diff
@@ -0,0 +1,80 @@
+# Initial layer configuration with Side-car
+[home](../README.md)
+* This Side-car's goal is to run over new geoserver and auto-generate layer that will serve entire suite of polygon parts features via WFS.
+* It watch geoserver REST-API endpoint availability and once detect so, process will be start.
+* Implemented with Nodejs and use fetch requests on several geoserver's rest APIs [/rest](https://docs.geoserver.org/main/en/user/rest/index.html).
+* Supports Javascript.
+
+## Main processes
+1. Start only if detect available geoserver endpoint.
+```javascript
+await checkGeoserverIsUp();
+```
+
+> [!CAUTION]
+> The method will run eternally until detect working endpoint of geoserver.
```

Comment:
detect -> it detects working geoserver endpoint.

Reply:
done